extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -150,10 +141,34 @@ export class ResourceTypePickerDialog extends DialogBase {
 				loadingText: localize('deploymentDialog.loadingRequiredTools', "Loading required tools information"),
 				showText: true
 			}).component();
+
+			const resourceComponents: azdata.Component[] = [];
+			if (this.getAllResourceTags().length !== 0) {
+				this._resourceTagsListView = this.createTagsListView();
+				resourceComponents.push(this._resourceTagsListView);
+			}
+			this._resourceSearchBox = view.modelBuilder.inputBox().withProperties({
+				placeHolder: localize('deploymentDialog.resourceSearchPlaceholder', "Filter resources..."),
```

Comment:
Please put all localized strings into the localizedConstants file - it makes it a lot easier to search and find strings to avoid duplication.